docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -83,6 +86,13 @@ The table below lists the data that the Bitbucket integration collects from your
     </td>
     <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
 </tr>
+<tr>
+    <td>Pull requests</td>
+    <td>
+        -
+    </td>
+    <td><a href="../../metrics/accelerate-changes/">Changes & Reviews dashboard</a>,<br/><a href="../../metrics/accelerate-wip/">Work in Progress dashboard</a></td>
+</tr>
```

Comment:
@afsmeira I'm updating the collected data table to include the new metrics for BB.
I'm adding this line to the table as it is in the GitHub integration, but it's missing the details about the mapping to Pulse data model.
Do we have any additional mapping for this line? Or does it make sense to merge this line with the previous one? (does the same mapping apply?)

Reply:
We don't do any mapping to the pull request data coming from Bitbucket. We simply cherry pick what we need and store it on our side to use for those dashboards.